### DISCUSSION TITLE
Export type "ClassValue" from clsx

### DISCUSSION THREAD

**Discussion by danciudev** [joe-bell/cva#172] [Ideas]
Hello, it would be interesting to export the **ClassValue** typo of the clsx library in the main package, to be able to use it together with tailwind-merge:

```
import type { ClassValue } from 'node_modules/class-variance-authority/dist/types';

// should be import type { ClassValue } from 'class-variance-authority';

import { cx } from 'class-variance-authority';
import { twMerge } from 'tailwind-merge';

export function cn(...inputs: ClassValue[]) {
	return twMerge(cx(inputs));
}
```
Also add it as an example in the documentation.

**Comment by joe-bell** [comment]
Hey! In `cva@1.0` you won't have to worry about this at all, and there will be a new API to extend `cx`

In the meantime, you can use the built-in `Parameters` type:

```ts
type ClassValue = Parameters<typeof cx>
```
